2024-07-08 21:27:42,053 - INFO - "GET / HTTP/1.1" 200 8980 - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\servers\basehttp.py - None
2024-07-08 21:27:43,752 - WARNING - Not Found: /news/ - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\log.py - None
2024-07-08 21:27:43,753 - WARNING - "GET /news/ HTTP/1.1" 404 4671 - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\servers\basehttp.py - None
2024-07-08 21:27:44,916 - WARNING - Not Found: /news/ - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\log.py - None
2024-07-08 21:27:44,916 - WARNING - "GET /news/ HTTP/1.1" 404 4671 - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\servers\basehttp.py - None
2024-07-12 06:04:15,619 - ERROR - Internal Server Error: /admin/news/comment/ - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\log.py - (<class 'AttributeError'>, AttributeError("'Comment' object has no attribute 'comment_post'"), <traceback object at 0x0000026533F7E6C0>)
Traceback (most recent call last):
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\db\models\options.py", line 681, in get_field
    return self.fields_map[field_name]
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: 'action_checkbox'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\utils.py", line 289, in lookup_field
    f = _get_non_gfk_field(opts, name)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\utils.py", line 322, in _get_non_gfk_field
    field = opts.get_field(name)
            ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\db\models\options.py", line 683, in get_field
    raise FieldDoesNotExist(
django.core.exceptions.FieldDoesNotExist: Comment has no field named 'action_checkbox'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\base.py", line 45, in render
    return super().render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\library.py", line 258, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 345, in result_list
    "results": list(results(cl)),
               ^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 321, in results
    yield ResultList(None, items_for_result(cl, res, None))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 312, in __init__
    super().__init__(*items)
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 217, in items_for_result
    f, attr, value = lookup_field(field_name, result, cl.model_admin)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\utils.py", line 298, in lookup_field
    value = attr(obj)
            ^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\options.py", line 999, in action_checkbox
    "aria-label": format_html(_("Select this object for an action - {}"), obj),
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\html.py", line 135, in format_html
    return mark_safe(format_string.format(*args_safe, **kwargs_safe))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\html.py", line 117, in conditional_escape
    return escape(text)
           ^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\functional.py", line 231, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\html.py", line 50, in escape
    return SafeString(html.escape(str(text)))
                                  ^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\news_project\news\models.py", line 110, in __str__
    return self.comment_post.author.user.username + '---' + str(self.id)
           ^^^^^^^^^^^^^^^^^
AttributeError: 'Comment' object has no attribute 'comment_post'
2024-07-12 06:04:15,667 - ERROR - "GET /admin/news/comment/ HTTP/1.1" 500 422021 - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\servers\basehttp.py - None
2024-07-15 20:43:10,101 - ERROR - Internal Server Error: /admin/news/comment/ - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\log.py - (<class 'AttributeError'>, AttributeError("'Comment' object has no attribute 'comment_post'"), <traceback object at 0x000002D310C87540>)
Traceback (most recent call last):
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\db\models\options.py", line 681, in get_field
    return self.fields_map[field_name]
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: 'action_checkbox'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\utils.py", line 289, in lookup_field
    f = _get_non_gfk_field(opts, name)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\utils.py", line 322, in _get_non_gfk_field
    field = opts.get_field(name)
            ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\db\models\options.py", line 683, in get_field
    raise FieldDoesNotExist(
django.core.exceptions.FieldDoesNotExist: Comment has no field named 'action_checkbox'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\base.py", line 45, in render
    return super().render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\library.py", line 258, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 345, in result_list
    "results": list(results(cl)),
               ^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 321, in results
    yield ResultList(None, items_for_result(cl, res, None))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 312, in __init__
    super().__init__(*items)
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 217, in items_for_result
    f, attr, value = lookup_field(field_name, result, cl.model_admin)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\utils.py", line 298, in lookup_field
    value = attr(obj)
            ^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\options.py", line 999, in action_checkbox
    "aria-label": format_html(_("Select this object for an action - {}"), obj),
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\html.py", line 135, in format_html
    return mark_safe(format_string.format(*args_safe, **kwargs_safe))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\html.py", line 117, in conditional_escape
    return escape(text)
           ^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\functional.py", line 231, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\html.py", line 50, in escape
    return SafeString(html.escape(str(text)))
                                  ^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\news_project\news\models.py", line 111, in __str__
    return self.comment_post.author.user.username + '---' + str(self.id)
           ^^^^^^^^^^^^^^^^^
AttributeError: 'Comment' object has no attribute 'comment_post'
2024-07-15 20:43:10,141 - ERROR - "GET /admin/news/comment/ HTTP/1.1" 500 422019 - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\servers\basehttp.py - None
2024-07-15 20:46:33,672 - ERROR - Internal Server Error: /admin/news/comment/ - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\log.py - (<class 'AttributeError'>, AttributeError("'Comment' object has no attribute 'comment_post'"), <traceback object at 0x0000014456BF24C0>)
Traceback (most recent call last):
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\db\models\options.py", line 681, in get_field
    return self.fields_map[field_name]
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: 'action_checkbox'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\utils.py", line 289, in lookup_field
    f = _get_non_gfk_field(opts, name)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\utils.py", line 322, in _get_non_gfk_field
    field = opts.get_field(name)
            ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\db\models\options.py", line 683, in get_field
    raise FieldDoesNotExist(
django.core.exceptions.FieldDoesNotExist: Comment has no field named 'action_checkbox'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\base.py", line 45, in render
    return super().render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\library.py", line 258, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 345, in result_list
    "results": list(results(cl)),
               ^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 321, in results
    yield ResultList(None, items_for_result(cl, res, None))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 312, in __init__
    super().__init__(*items)
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 217, in items_for_result
    f, attr, value = lookup_field(field_name, result, cl.model_admin)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\utils.py", line 298, in lookup_field
    value = attr(obj)
            ^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\options.py", line 999, in action_checkbox
    "aria-label": format_html(_("Select this object for an action - {}"), obj),
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\html.py", line 135, in format_html
    return mark_safe(format_string.format(*args_safe, **kwargs_safe))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\html.py", line 117, in conditional_escape
    return escape(text)
           ^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\functional.py", line 231, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\html.py", line 50, in escape
    return SafeString(html.escape(str(text)))
                                  ^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\news_project\news\models.py", line 110, in __str__
    return self.comment_post.author.user.username + '---' + str(self.id)
           ^^^^^^^^^^^^^^^^^
AttributeError: 'Comment' object has no attribute 'comment_post'
2024-07-15 20:46:33,699 - ERROR - "GET /admin/news/comment/ HTTP/1.1" 500 422019 - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\servers\basehttp.py - None
2024-07-15 20:56:22,304 - ERROR - Internal Server Error: /admin/news/comment/ - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\log.py - (<class 'AttributeError'>, AttributeError("'Comment' object has no attribute 'comment_post'"), <traceback object at 0x0000020B0D202AC0>)
Traceback (most recent call last):
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\db\models\options.py", line 681, in get_field
    return self.fields_map[field_name]
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: 'action_checkbox'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\utils.py", line 289, in lookup_field
    f = _get_non_gfk_field(opts, name)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\utils.py", line 322, in _get_non_gfk_field
    field = opts.get_field(name)
            ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\db\models\options.py", line 683, in get_field
    raise FieldDoesNotExist(
django.core.exceptions.FieldDoesNotExist: Comment has no field named 'action_checkbox'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\base.py", line 45, in render
    return super().render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\library.py", line 258, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 345, in result_list
    "results": list(results(cl)),
               ^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 321, in results
    yield ResultList(None, items_for_result(cl, res, None))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 312, in __init__
    super().__init__(*items)
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 217, in items_for_result
    f, attr, value = lookup_field(field_name, result, cl.model_admin)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\utils.py", line 298, in lookup_field
    value = attr(obj)
            ^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\contrib\admin\options.py", line 999, in action_checkbox
    "aria-label": format_html(_("Select this object for an action - {}"), obj),
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\html.py", line 135, in format_html
    return mark_safe(format_string.format(*args_safe, **kwargs_safe))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\html.py", line 117, in conditional_escape
    return escape(text)
           ^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\functional.py", line 231, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\html.py", line 50, in escape
    return SafeString(html.escape(str(text)))
                                  ^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\news_project\news\models.py", line 111, in __str__
    return self.comment_post.author.user.username + '---' + str(self.id)
           ^^^^^^^^^^^^^^^^^
AttributeError: 'Comment' object has no attribute 'comment_post'
2024-07-15 20:56:22,329 - ERROR - "GET /admin/news/comment/ HTTP/1.1" 500 422019 - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\servers\basehttp.py - None
2024-07-15 21:40:40,881 - ERROR - Internal Server Error: / - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\log.py - (<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'set_language' not found. 'set_language' is not a valid view function or pattern name."), <traceback object at 0x00000226A61ED2C0>)
Traceback (most recent call last):
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\urls\resolvers.py", line 851, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'set_language' not found. 'set_language' is not a valid view function or pattern name.
2024-07-15 21:40:40,892 - ERROR - "GET /?page=1 HTTP/1.1" 500 182808 - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\servers\basehttp.py - None
2024-07-16 07:10:32,145 - ERROR - Internal Server Error: / - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\log.py - (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError('Unknown argument for \'trans\' tag: \'D3\'. The only options available are \'noop\', \'context\' "xxx", and \'as VAR\'.'), <traceback object at 0x00000298528AB840>)
Traceback (most recent call last):
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 134, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 131, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 109, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\templatetags\i18n.py", line 443, in do_translate
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Unknown argument for 'trans' tag: 'D3'. The only options available are 'noop', 'context' "xxx", and 'as VAR'.
2024-07-16 07:10:32,174 - ERROR - "GET / HTTP/1.1" 500 218580 - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\servers\basehttp.py - None
2024-07-23 21:39:11,932 - ERROR - Internal Server Error: /news/search/ - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\log.py - (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Could not parse the remainder: ''<input' from ''<input'"), <traceback object at 0x00000253D6D28700>)
Traceback (most recent call last):
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\templatetags\i18n.py", line 403, in do_translate
    message_string = parser.compile_filter(bits[1])
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 597, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 698, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: ''<input' from ''<input'
2024-07-23 21:39:11,951 - ERROR - "GET /news/search/?title__icontains=%D0%A2%D1%80%D0%B5%D1%82%D1%8C%D1%8F%D0%BA%D0%BE%D0%B2%D0%BA%D0%B5&added_after= HTTP/1.1" 500 207760 - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\servers\basehttp.py - None
2024-07-23 21:41:09,206 - ERROR - Internal Server Error: /news/search/ - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\utils\log.py - (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Could not parse the remainder: ''' from 'Find''"), <traceback object at 0x00000253D6F85E40>)
Traceback (most recent call last):
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\templatetags\i18n.py", line 403, in do_translate
    message_string = parser.compile_filter(bits[1])
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 597, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\template\base.py", line 698, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: ''' from 'Find''
2024-07-23 21:41:09,219 - ERROR - "GET /news/search/?title__icontains=%D0%A2%D1%80%D0%B5%D1%82%D1%8C%D1%8F%D0%BA%D0%BE%D0%B2%D0%BA%D0%B5&added_after= HTTP/1.1" 500 208453 - D:\Documents\git_test_clone\NewsProject2\venv\Lib\site-packages\django\core\servers\basehttp.py - None
